_pkgdown.yml

```diff
@@ -1,3 +1,17 @@
 url: https://gilead-biostats.github.io/gsm.app/
 template:
   bootstrap: 5
+
+navbar:
+  structure:
+    left: [intro, reference, articles, tutorials, news]
+    right: [search, github]
+
+articles:
+- title: Getting Started
+  navbar: ~
+  contents:
+  - data-preparation
+- title: Advanced Topics  
+  contents:
+  - plugins
```

Comment:
```suggestion
```
Until we have a very good reason to do so, don't specify what goes where. Once we start doing this, we have the extra step of adding a thing here when we add a vignette. If we leave this off, pkgdown will simply include every vignette, without us having to do an extra step.